gradle/libs.versions.toml

```diff
@@ -64,18 +64,18 @@ managed-jaeger = "1.6.0"
 managed-jakarta-annotation-api = "2.0.0"
 managed-jackson = "2.12.4"
 managed-javax-annotation-api = "1.3.2"
-managed-jcache = "1.1.0"
+managed-jcache = "1.1.1"
 managed-jna = "5.7.0"
 managed-jsr305 = "3.0.2"
-managed-kafka = "2.8.0"
-managed-ktor = "1.6.1"
-managed-lettuce = "6.1.1.RELEASE"
+managed-kafka = "2.8.1"
+managed-ktor = "1.6.7"
+managed-lettuce = "6.1.6.RELEASE"
```

Comment:
These are managed by micronaut kafka, kotlin and redis modules

Reply:
Micronaut Kotlin, Kafka and Redis should provide BOMs